src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -1,5 +1,7 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+// Despite its name, this file IS NOT (yet) automatically generated.
```

Comment:
I see you have figured out my secret to preventing people from changing my code.